### PR TITLE
Updates for seeing calculation.

### PR DESCRIPTION
**Pull request by mareuter**

### CODE REVIEW

File: python/lsst/sims/operations/Simulator.py [hunk 1]
```diff
@@ -105,6 +108,8 @@ def __init__(self, lsstDB, obsProfile, sessionID, nRun, seeingEpoch, simStartDay
         self.telSeeing = telSeeing
         self.opticalDesSeeing = opticalDesSeeing
         self.cameraSeeing = cameraSeeing
+        self.scaleToNeff = scaleToNeff
+        self.atmNeffFactor = atmNeffFactor
```
Comment (rhiannonlynne) [inline python/lsst/sims/operations/Simulator.py:112]: Why are the scaleToNeff and atmNeffFactor (and other parameters) stored here in the Simulator class, when they are also stored in the Filters class (of which there is one belonging to the Simulator class, as far as I can tell)?

**Comment by rhiannonlynne** [comment]
I like the unit test. One question (in line comments).

File: python/lsst/sims/operations/Simulator.py (hunk 1 above)
Comment (mareuter) [inline python/lsst/sims/operations/Simulator.py:112]: Just following the convention in the code of how to get parameters from one class to another. Doing it the right way would mean tearing a lot of things apart.

Comment (rhiannonlynne) [inline python/lsst/sims/operations/Simulator.py:112]: Fair enough.

**Comment by rhiannonlynne** [comment]
Looks good to me. In the JIRA ticket, Kem said he was good with this too and had checked some examples.

**Pull request merged**